content/networking/private-endpoints/dns-pe-concepts.md

```diff
@@ -19,10 +19,12 @@ A common challenge is that organizations wish to use their own zone names for th
 
 ## DNS Options in Azure
 
-There are two primary options to consider for creating DNS records for your Private Endpoint resources:
+There are three primary options to consider for creating DNS records for your Private Endpoint resources:
 
 - **Private DNS Zones:** Private DNS Zones are Azure's native Private DNS offering and integrate with the default Azure-provided DNS service. Using Private DNS Zones with your Private Endpoints enables automated DNS record management for your Private Endpoints, improving scale and ease of management.
 - **Custom DNS servers:** Custom DNS servers are when you bring your own DNS servers to Azure point your DNS clients to them. Commonly, these are Windows Domain Controllers hosting an extension of your on-prem DNS services. Custom DNS servers can be combined with Private DNS zones to provide both the consistency of your extended DNS service and the flexibility of Private DNS Zones. Alternatively, you can skip Private DNS Zones altogether and create 'A' DNS records for your Private Endpoints directly in your Custom DNS server--this is usually not recommended because it scales poorly without significant automation investment.
+  
+- **Azure DNS Private resolver:** Azure DNS Private Resolver is a new fully managed service that enables you to query Azure DNS private zones from an on-premises environment and vice versa without deploying VM based DNS servers. Azure DNS Private Resolver requires an Azure Virtual Network. When you create an Azure DNS Private Resolver inside a virtual network, one or more inbound endpoints are established that can be used as the destination for DNS queries. The resolver's outbound endpoint processes DNS queries based on a DNS forwarding ruleset that you configure. DNS queries that are initiated in networks linked to a ruleset can be sent to other DNS servers.
```

Comment:
I think this may be the wrong place for it - this section talks about creating the DNS records themselves, which private resolver doesn't let you do.  Maybe including this text in this section: https://github.com/Azure/FTALive-Sessions/blob/main/content/networking/private-endpoints/dns-pe-scenarios.md

Where it adds to the text about the DNS servers in the hub?